spec/v0-1.md

```diff
@@ -111,6 +120,18 @@ The expression `(eval <exp>)` or `(eval <exp> <env>)` first evaluates the expres
 
 While `eval` is part of core Lurk, it is not part of the initial `lurk-rs` release.
 
+## Numeral Parsing 
+
+The release specs of `lurk-rs` has diverged from core Lurk. The following parsing rules are only supported by `lurk-rs`. 
```

Comment:
Ideally, this spec is the source of truth. We shouldn't be discussing `lurk-rs` at all. For present purposes, it would be better for the spec to agree with `lurk-rs` and for divergences in the `lurk` repo to be treated as bugs.